src/garage/np/algos/cem.py

```diff
@@ -81,6 +77,8 @@ def train(self, runner):
             The average return in last epoch cycle.
 
         """
+        self.n_samples = runner.train_args.n_epoch_cycles
```

Comment:
so this behavior is surprising -- can you instead just make it a hyperparameter of the algorithm object? i don't think this is part of the runner interface.

Reply:
Are you suggesting putting this to `BatchPolopt` class?

Reply:
@ryanjulian I didn't get your point. My understanding is for CEM and CMA_ES, we set `n_samples` by defining `n_epoch_cycles` via runner interface. Would you clarify?

Reply:
I am proposing that we make n_samples a constructor argument and don't use `n_epoch_samples` to define `n_samples`, because I think that's confusing (they are not the same thing).